.dependabot/config.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
```

Comment:
For this file, I would maybe instead use the suggested location from #308: `.github/dependabot.yml` instead.
In this way we're also not removing the `.ci` folder in favor another new folder :)
Otherwise it should be fine :)

Reply:
Didn't even notice that.